userspace/libscap/engine/modern_bpf/scap_modern_bpf.c

```diff
@@ -29,6 +31,9 @@ limitations under the License.
 #include <sys/utsname.h>
 #include "ringbuffer/ringbuffer.h"
 
+#include <linux/bpf.h>
+#include <bpf/libbpf.h>
```

Comment:
probably the right place to perform these checks is `libpman` the modern bpf engine doesn't use `libbpf` explicitly